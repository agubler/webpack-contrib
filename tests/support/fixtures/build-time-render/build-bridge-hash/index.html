<html>
	<head>
		<link href="main.css" rel="stylesheet">
	</head>
	<body>
		<div id="app"></div>
		<script type="text/javascript" src="bootstrap.abcdefghij0123456789.bundle.js"></script>
		<script type="text/javascript" src="main.0123456789abcdefghij.bundle.js"></script>
	</body>
</html>
