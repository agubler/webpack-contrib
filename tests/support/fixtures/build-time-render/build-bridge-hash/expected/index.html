<html>
	<head>
	<style>span{width:100%}</style></head>
	<body>
		<div id="app"><div>hello world a</div></div>
		<script type="text/javascript" src="bootstrap.c587a9b6b98529303de8.bundle.js"></script>
		<script type="text/javascript" src="main.60ebb7634ba23af82ec9.bundle.js"></script>
	</body>
</html>
